reference/shaders-msl-no-opt/frag/volatile-helper-invocation.msl23.spv16.frag

```diff
@@ -8,18 +10,24 @@ struct main0_out
     float FragColor [[color(0)]];
 };
 
+static inline __attribute__((always_inline))
+void func(thread float& FragColor, volatile thread bool& gl_HelperInvocation)
```

Comment:
volatile thread seems like it's asking for trouble with spilling. volatile thread is somewhat meaningless. I know this comes up because in later SPIR-V discard + helper lane is done through volatile StorageClassInput.

Reply:
Do you think we should just avoid `volatile` on `thread` variables in general?